test/WebJobs.Script.Tests/TestScripts/Node/HttpTriggerExpressApi/index.js

```diff
@@ -0,0 +1,20 @@
+﻿var util = require('util');
+
+module.exports = function (context, req) {
+    context.log('Node.js HttpTrigger function invoked.');
+
+    context.res.status(200)
```

Comment:
I'm already adding captured route params to `req.params` [here](https://github.com/Azure/azure-webjobs-sdk-script/blob/dev/src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs#L479)